app/views/datasets/show.html.erb

```diff
@@ -75,7 +75,7 @@
             <div role="note" aria-label="Publisher information" class="panel panel-border-narrow text">
               <p><%= t('datasets.publishers.information') %></p>
             </div>
-            <p><%= link_to t('datasets.publishers.button'), "/dataset/edit/#{@dataset.legacy_name}", role: 'button', class: 'button', rel: %w(nofollow) %></p>
+            <p><%= link_to t('datasets.publishers.button'), "#{Plek.find("ckan")}/user/login", role: 'button', class: 'button', rel: %w(nofollow) %></p>
```

Comment:
Wait a second... Find is hosted on PaaS, isn't it, so `Plek` won't work.

Reply:
Oh well, `ckan.publishing.service.gov.uk` it is because Find doesn't have a non-production.